CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 - Fixed same line handling of singleline return bodies of functions ([#514](https://github.com/HaxeCheckstyle/haxe-formatter/issues/514))
 - Fixed missing space when returning an arrow function, fixes [#513](https://github.com/HaxeCheckstyle/haxe-formatter/issues/513) ([#514](https://github.com/HaxeCheckstyle/haxe-formatter/issues/514))
 - Fixed conditionals in return types, fixes [#446](https://github.com/HaxeCheckstyle/haxe-formatter/issues/446) ([#515](https://github.com/HaxeCheckstyle/haxe-formatter/issues/515))
+- Fixed null pointer issues in MarkWhitespace ([#521](https://github.com/HaxeCheckstyle/haxe-formatter/issues/521))
+- Fixed path handling in Hashlink 1.1 ([#521](https://github.com/HaxeCheckstyle/haxe-formatter/issues/521))
```

Comment:
You mean 1.10? 1.1 is really old. ;)

Reply:
Besides, why would that path handling not work on HL? HL bug?

Reply:
yes 1.1 (plus a patch for readdir from 1.3), not 1.10
as far as I could figure out 1.1 is the last version that works with Haxe 3.4.7

paths would have a trailing newline and concatenation looked like `path//////////////filename`.

Also Hashlink 1.1 is very picky when it comes to Nullpointers so it's a good test :)